images/build/debian-iptables/buster/Dockerfile

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 ARG BASEIMAGE
+ARG GORUNNERIMAGE
```

Comment:
Hmm, now that I look at how this is getting used in k/k master I realize that a similar change is probably also needed for bullseye/Dockerfile, right?

Let me try to come up with a PR...